service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsSubmittedControllerTest.java

```diff
@@ -188,29 +188,25 @@ void init() {
         given(documentDownloadService.downloadDocument(ORDER.getBinaryUrl())).willReturn(ORDER_BINARY);
         given(uploadDocumentService.uploadPDF(ORDER_BINARY, ORDER.getFilename()))
             .willReturn(ORDER_DOCUMENT);
-        given(documentService.createCoverDocuments(any(), any(), eq(REPRESENTATIVE_WITH_POST_PREFERENCE.getValue())))
-            .willReturn(DocmosisDocument.builder().bytes(COVERSHEET_REPRESENTATIVE_BINARY).build());
         given(documentService.createCoverDocuments(any(), any(), eq(OTHER_REP_BY_POST.getValue())))
             .willReturn(DocmosisDocument.builder().bytes(COVERSHEET_OTHER_REPRESENTATIVE_BINARY).build());
-        given(uploadDocumentService.uploadPDF(COVERSHEET_REPRESENTATIVE_BINARY, "Coversheet.pdf"))
-            .willReturn(COVERSHEET_REPRESENTATIVE);
         given(uploadDocumentService.uploadPDF(COVERSHEET_OTHER_REPRESENTATIVE_BINARY, "Coversheet.pdf"))
             .willReturn(COVERSHEET_OTHER_REPRESENTATIVE);
         given(sendLetterApi.sendLetter(any(), any(LetterWithPdfsRequest.class)))
-            .willReturn(new SendLetterResponse(LETTER_1_ID))
-            .willReturn(new SendLetterResponse(LETTER_2_ID));
+            .willReturn(new SendLetterResponse(LETTER_1_ID));
     }
 
     @Test
     void submittedEventShouldNotifyHmctsAdminAndRepresentativesWhenCtscToggleIsDisabled() {
         given(featureToggleService.isServeOrdersAndDocsToOthersEnabled()).willReturn(true);
 
+        List<Element<Respondent>> respondents = List.of(element(RESPONDENT_WITH_DIGITAL_REP),
```

Comment:
I think these constants could be `Element<Respondent>` instead of `Respondent`.
I'll write the reason on the comment below.